### PR TITLE
Add message emoji getter

### PR DESCRIPTION
**Pull request by devsnek**
**Please describe the changes this PR makes and why it should be merged:**
adds Message#emojis as a getter to get emojis in the message

**Semantic versioning classification:**  
- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by iCrawl** [comment]
Not too sure about this.

**Comment by bdistin** [comment]
1+ to crawl's comment

**Comment by SnazzyPine25** [comment]
can we bring up #1691 since the regex is in this pr

File: src/structures/Message.js [hunk 1]
```diff
@@ -248,6 +248,35 @@ class Message {
...
+  /**
+   * Get emojis in this message's content
+   * @type {MessageEmoji[]}
+   * @readonly
+   */
+  get emojis() {
+    const matches = [];
+    let match;
+    // eslint-disable-next-line no-cond-assign
+    while (match = Constants.UnicodeEmojiRegex().exec(this.content)) {
```
Comment (1Computer1) [inline src/structures/Message.js]: The regex should be cached or else it will keep matching the same emoji.

File: src/structures/Message.js [hunk 2]
```diff
@@ -248,6 +248,35 @@ class Message {
...
+      matches.push({ emoji: match[0], index: match.index });
+    }
+    const CustomRe = /<:(\w+):(\d+)>/g;
```
Comment (1Computer1) [inline src/structures/Message.js]: Lowercase `c` please

**Pull request closed without merging**